### PR TITLE
Add skyler bio page

### PR DESCRIPTION
**Pull request by skylerbasco**
https://trello.com/c/2vjjhVdz/2-skyler-bio-page

Screen Shot 2022-01-11 at 10 54 50 AM

You can press the play button at the top to play "Three is the Magic Number" from Spider-Man NWH, click buttons to see my projects/3D Bitly logo, see animated emojis, and submit a message through the contact form.

### CODE REVIEW

**Comment by skylerbasco** [comment]
https://trello.com/c/2vjjhVdz/2-skyler-bio-page

Screen Shot 2022-01-11 at 10 54 50 AM

You can press the play button at the top to play "Three is the Magic Number" from Spider-Man NWH, click buttons to see my projects/3D Bitly logo, and submit a message through the contact form.

File: src/styles/2022_team_styles/_skyler-bio.scss [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+    text-align: center;
+}
+
+.main-container {
+    display: table;
+    overflow: hidden;
+    min-height: 4200px; /* USE THIS TO ADJUST FIXED CONTAINER HEIGHT */
+    padding-left: 250px;
+    padding-right: 250px;
+   /* min-height: 1300px !important;
```
Comment (angelouis) [inline src/styles/2022_team_styles/_skyler-bio.scss:83]: If you want you can delete any unused code :)

File: src/styles/2022_team_styles/_skyler-bio.scss [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    display: block;
+    text-align: center;
+    padding: 0px;
+    width: 200px;
+    height: auto;
+    margin: auto;
+}
+
+
+/* CONTACT SECTION */
```
Comment (angelouis) [inline src/styles/2022_team_styles/_skyler-bio.scss:263]: I like how you used the comments to organize your code :)

File: src/styles/_footer.scss [hunk 3]
```diff
@@ -2,10 +2,12 @@
 
 .footer {
   background-color: #063B6F;
+
```
Comment (angelouis) [inline src/styles/_footer.scss:5]: You can remove the extra spaces if you want :)

File: src/wintern-bios/2022/SkylerComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,176 @@
+import React from 'react';
```
Comment (angelouis) [inline src/wintern-bios/2022/SkylerComponent.jsx:1]: Great naming conventions :)

File: src/wintern-bios/2022/TeamLandingPage2022.jsx [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+import SkylerComponent from './SkylerComponent.jsx';
+
 
 const TeamLandingPage2022 = () => {
+
+
   return (
     <div>
         OMG. IT'S A BLANK WEBSITE!
+        { <SkylerComponent /> }
```
Comment (angelouis) [inline src/wintern-bios/2022/TeamLandingPage2022.jsx:11]: You can remove the curly braces if you want :)

**Review by angelouis — APPROVED** [review]
Your website looks so amazing! It's your choice if you want to make the stylistic changes, but overall it looks great :)

**Comment by angelouis** [comment]
QA Passed :)

**Pull request merged**